sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,18 @@
+select "genres"."name",
+      count(*) as "total"
+  from "genres"
+join "filmGenre" using ("genreId")
+join "castMembers" using ("filmId")
+join "films" using ("filmId")
+join "actors" using ("actorId")
+  where "actors"."firstName" = 'Lisa'
+    and "actors"."lastName" = 'Monroe'
+group by "genres"."name";
```

Comment:
It would be best to group by genreId to correctly handle the unlikely case two where genres happen to have the same name.